### DISCUSSION TITLE
Modifying the description of ASA Stats in the meta tag of the website

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#57] [General]
The discussion is moved from the [related issue](https://github.com/asastats/channel/issues/36).

----- 

https://discord.com/channels/906917846754418770/908054330265960478/937756668601696368

> > ASA Stats shows the current value of all assets located in any Algorand account.
> This is not true, as it shows all assets associated with an Algorand account regardless of its location.

https://discord.com/channels/906917846754418770/908054330265960478/937799154376278026
> This is the description in the meta tag of the site. It's what shows up in search engines and in previews here in Discord and probably other chat platforms when pasting links.

> Maybe this needs to be expanded further to really stand out from the others.

> ASA Stats shows the current value of all assets associated with any Algorand account, including assets that are locked away in staking pools, liquidity pools, lending/borrowing, etc..

> I'm not 100% on the wording but I think including this kind of info will be useful and should be listed. This is what ASA Stats is all about, it should be up front.

-----

The following provide an introduction to Meta Tags and propose an update. 

Information Resources: [Meta Description](https://moz.com/learn/seo/meta-description#:~:text=Meta%20descriptions%20can%20technically%20be,between%2050%20and%20160%20characters), [How to create the right meta description](https://yoast.com/meta-descriptions/), [SEO basics: What is a search term?](https://yoast.com/search-term/)

### What is a meta description?
The meta description is an HTML tag you can set for a post or page of your website. In it, you can describe what your page is about.

### Characteristics of a good meta description

- **Characters Count**: The right length doesn’t really exist. [Recommend](https://moz.com/learn/seo/meta-description#:~:text=Meta%20descriptions%20can%20technically%20be,between%2050%20and%20160%20characters.) descriptions between 50 and 160 characters 
- **Use active voice and make it actionable** 
- **Include a call-to-action**
- **Show specifications, where possible** Target visitors specifically looking for that product
- **Make sure it matches the content of the page**: [ASA Stats About page](https://www.asastats.com/about/)
- **Use your focus keyword** If the search keyword matches a part of the text in the meta description, Google will be more inclined to use it and highlight it in the search results.

### [Google Keyword Planner](https://ads.google.com/aw/keywordplanner/ideas/new?ocid=213532213&euid=221073094&__u=8812726806&uscid=213532213&__c=2319813837&authuser=0&sf=kp&subid=gb-en-ha-aw-sk-c-bau%21o3~Cj0KCQjw37iTBhCWARIsACBt1Iz14kT9COPsCWUBIrEmdUoO2ZRCsUwWblzjB3sAVE2vnxsgfSRHCNYaAjn1EALw_wcB~112305137240~kwd-58438391677~11199860533~467745007104)
Average monthly searches, English Language, USA, Canada, Europe, China
- NFT: 1M – 10M
- ASA: 10K – 100K
- Price: 10K – 100K
- portfolio tracker: 1K – 10K

### Proposed update
ASA Stats evaluates and presents ASA and NFT price information on the Algorand blockchain. Protfolio Tracker, API and Feature-Driven Development. Check it out.

Check [word density and characters](https://www.charactercountonline.com/) 
Characters count: 158 | Words: 23

-----

**An edit to the proposed update:** 

ASA Stats evaluates and displays ASA and NFT price information on the Algorand blockchain.

ASA Stats displays the current value of all assets in an Algorand account, including assets held in staking pools, liquidity pools, lending/borrowing, among others.

ASA Stats offers Portfolio Tracker, API, and feature-driven development

-----

> **An edit to the proposed update:**
> 
> ASA Stats evaluates and displays ASA and NFT price information on the Algorand blockchain.
> 
> ASA Stats displays the current value of all assets in an Algorand account, including assets held in staking pools, liquidity pools, lending/borrowing, among others.
> 
> ASA Stats offers Portfolio Tracker, API, and feature-driven development

Please enter any term in the search box at Google (https://www.google.com) and scroll down the page that opens.

Every row is consisting of three items:

a) link 

 That's "https://www.asastats.com" in our case

b) Title

That's (currently) "ASA Stats" for the root page of our site

c) Description

That's what we need. https://github.com/asastats/channel/issues/36#issuecomment-1114328374

-----

**Comment by Sammyspeed75** [comment]
@ipaleka @SCN9A, based on previous comments, and after taking the highlighted keywords and other prerequisites into consideration. I came up with another draft for the meta description: 

The ASA Stats **portfolio tracker** displays the **current value of ALL assets (ALGO, ASA, NFT)** associated with your **Algorand** account in real time, including assets held in **staking pools, liquidity pools, lending and borrowing protocols,** among others. 

Does this suit the purpose better?

**Reply by ipaleka** [reply]
> Does this suit the purpose better?

Nope. https://github.com/asastats/channel/issues/36#issuecomment-1114328374

**Comment by ipaleka** [comment]
@SCN9A [proposed update](https://github.com/asastats/channel/issues/36#issuecomment-1114328374) has been deployed.
